ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,156 @@
 <!--
-  -  NextCloud rules
-  -  Author: banditopazzo.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
 <!--
-- It's needed to enable audit in Nextcloud if you want full log
```

Comment:
The English is weird here.

This looks better
"Enabling audit in your Nextcloud system is required to get the full log message"

It is not clear what the full log means here. Is there a way to collect Nextcloud with no full log? A better explanation is needed.

Reply:
It's not clear for me either, I am not a NextCloud user or specialist.

Reply:
Improved decription. Perhaps mroe work is needed